.rubocop.yml

```diff
@@ -1,33 +1,129 @@
-inherit_gem:
-  rubocop-rootstrap:
-    - config/rails_edge.yml
+inherit_from:
+  - .rubocop_todo.yml
 
 inherit_mode:
   merge:
     - Exclude
 
+require:
+  - rubocop-factory_bot
+  - rubocop-performance
+  - rubocop-rails
+  - rubocop-rake
+  - rubocop-rspec
+
 AllCops:
+  NewCops: enable
```

Comment:
Using this config, new cops ("pending" cops) will be enabled when we upgrade the gem. The other option would be to use `disable` and manually enable the new ones.
https://docs.rubocop.org/rubocop/versioning.html#pending-cops

I'm ok keeping it as `enable`. I think that will help us to be aware of the new cops and understand them in a better way so we can decide if we really want it or not. 

I would also like to hear the group opinion here 🙂

Reply:
I think I'm good with the enable option, especially since it's a base app